### ISSUE TITLE
When finetuning on SROIE

### ISSUE THREAD

**Issue by TaekyungKi** [applicaai/lambert#5]
Hi. Thank you for interesting work and releasing the code. Can i ask some questions when finetuing on SROIE?
1. As SROIE is a dataset of reciept, the number of valid token is pretty small compared to 512. Did you still use 512 for sequence length? I think too many padding tokens may give noise to model since the model initialized with RoBERTa.

2. Did you use BIO tags in word-level? SROIE key-value pair is not word level, rather line level. Let me give some example. If given key 'company', value is "STARBUCKS Store #10208" and OCR is give as splitted strings "STARBUCKS Store", "#10208". 
Some people tags B-Company to the first OCR string and I-Company to the second OCR string, and then tokenizes it, so that the result seems like:
("STAR", "B-Company"), ("BUCKS", "B-Company"), ("Store", **"B-Company"**), ("#", "I-Company"), ("10208", "I-Company").
But i think BIO should be tagged in word level. So the results should be 
("STAR", "B-Company"), ("BUCKS", "B-Company"), ("Store", "**I-Company**"), ("#", "I-Company"), ("10208", "I-Company").
In short, only the first word 'STARBUCKS" is tagged as "B-Company" and the others are "I-Company".
It may be a minor issue, but SROIE contains complicate address or company name and i just want to check the right way. Can you tell me which one did you use?
In short, only the first word 'STARBUCKS" is tagged as "B-Company" and the others are "I-Company".
+ Or, did you use token-level tags?
("STAR", "**B-Company**"), ("BUCKS", "I-Company"), ("Store", "I-Company"), ("#", "I-Company"), ("10208", "I-Company").

Thank you for reading!
Best.

**Comment by tstanislawek** [comment]
ad 1) Good point, but we run our experiments with default 512 BPE tokens count 
ad 2)  We tested BIO and IO tag schema (on BPE token level) and decided to used IO for final solution since in this case we did not have to many tokens which are next to each other (only few examples for TOTAL entity)

**Comment by TaekyungKi** [comment]
Hi thanks for advising. Can I ask some more questions? In my setup, the model is suffering from finding "total" entity, on the other hand it easily finds the other entities. I guess.. "total" entity in SROIE has diversity, e.g. 100.10, 19.00, SR 8.00, $10.10... and it can appear several times in single receipt image.

So the questions are... 
1) Did your model also suffer from finding 'total' entity?
2) Did you use duplicate labels for 'total' entity?

Best.

**Comment by tstanislawek** [comment]
ad 1) You can see results for each model on the contest page (https://rrc.cvc.uab.es/?ch=13&com=evaluation&view=method_samples&task=3&m=82433&p=1&l=1) 
ad 2) We labeled in train set all tokens that suit golden value pattern
